src/transformers/training_args.py

```diff
@@ -569,7 +569,9 @@ class TrainingArguments:
                     Will use gradient checkpointing over each nested XLA FSDP wrapped layer. This setting can only be
                     used when the xla flag is set to true, and an auto wrapping policy is specified through
                     fsdp_min_num_params or fsdp_transformer_layer_cls_to_wrap.
-
+        tp_size (`int`, *optional*):
+            Use tp_size to enable PyTorch tensor parallelism. Set a value greater than 1 to activate TP. The same is
+            used to prepare device mesh internally. Requires accelerate>1.3.0.
```

Comment:
we should explain that this uses the models' `base_tp_plan` and is not available for all models etc!

Reply:
@ArthurZucker added this fix thank you